### ISSUE TITLE
Loader frezee when login and enters home page

### ISSUE THREAD

**Issue by vishy0001** [Anthony-Shoshi/Flutter-Udemy-Clone#1]
Unable to go head on home page loader is coming which frezee the screen can not go ahead.
![udemy](https://user-images.githubusercontent.com/95673466/170887251-0773a335-450f-46fc-b7fb-bfa3ba50e50a.png)

**Comment by Anthony-Shoshi** [comment]
Have you checked your debug console? is there any error? If so then please share the error message here.

**Comment by vishy0001** [comment]
NoSuchMethodError: The getter 'authentication' was called on null.
E/flutter (16544): Receiver: null

can you upload firebase schema which is used here so that I can check how data is coming??

**Comment by vishy0001** [comment]
can you please provide solution for this??

**Comment by Anthony-Shoshi** [comment]
Of course. I will check and if there any error, I will solve and push the updated code soon.
